### ISSUE TITLE
Species richness of mammal guilds

### ISSUE THREAD

**Issue by JamesDMSpeed** [JamesDMSpeed/PhyloTroph#3]
Herbivores
Carnivores
Insectivores
Omnivores

Start by making historgrams of Norwegian mammals (% plants, % vertebrate, % invertebrates)

**Comment by kjerstrs** [comment]
I struggle a bit to understand how we are going to use the histograms, and what we want them to show.  Is the purpose to make 3 different histograms based on the diet columns in the Trait_Data? E.g. one histogram based on the column Diet.plant in the Trait_Data?

**Comment by JamesDMSpeed** [comment]
Yes that is correct. We want to see the distribution of different diet types across the Norwegian mammals. I have just pushed a csv listing the relevant species that you can use to subset the global dataset

**Comment by JamesDMSpeed** [comment]
Great job with the histograms @kjerstrs ! I have made a small change to limit them to the terrestrial species (I also removed seals too), and played a bit with different definitions of omnivores that we can discuss tomorrow.
